File: Java/Cpu.java
public class Cpu{
    
    private String nama; //deklarasi Attribute
    private int harga; //deklarasi Attribute

    // Constructor 
    Cpu(){

    }
    
    Cpu(String nama, int harga){
        this.nama = nama;
        this.harga = harga;
    }
    // Get dan Set
    public String getNama() {
        return nama;
    }
    public void setNama(String nama) {
        this.nama = nama;
    }
    public int getHarga() {
        return harga;
    }
    public void setHarga(int harga) {
        this.harga = harga;
    }
   
    //keluaran
    public void printCpu(){
        System.out.println("---- CPU ----");
        System.out.println("Name: " + this.nama);
        System.out.println("Price: " + this.harga);
    }
}

File: C++/cpu.cpp
#include <iostream>
#include <string>
using namespace std;
class cpu{
    
    private:
        string nama; //deklarasi Attribute
        int harga; //deklarasi Attribute
    public:
        //Constructor
        cpu(){

        }
    
        cpu(string name, int price){
            this->nama = name;
            this->harga = price;
        }
        // get dan Set
        void setNama(string name){
            this->nama = name;
        }
        string getNama(){
            return this->nama;
        }
        void setHarga(int price){
            this->harga = price;
        }
        int getHarga(){
            return this->harga;
        }
        //keluaran
        void printCpu(){
            cout << "---- CPU ----" << endl;
            cout << "Name: " << this->nama << endl;
            cout << "Price: " << this->harga << endl;
        }
        
        ~cpu(){}
};